experimental/devices/mfrc522/mfrc522.go

```diff
@@ -181,7 +188,11 @@ func (r *Dev) Init() error {
 		return err
 	}
 
-	if err := r.devWrite(int(commands.RFCfgReg), byte(r.antennaGain)<<4); err != nil {
+	r.aMu.Lock()
+	err := r.devWrite(int(commands.RFCfgReg), byte(r.antennaGain)<<4)
```

Comment:
it's faster to do:

```
  r.aMu.Lock()
  gain := r.antennaGain
  r.aMu.Unlock()
  if err := r.devWrite(int(commands.RFCfgReg), byte(gain)<<4) {
    return err
  }
```

otherwise the call itself is locking